api/server.go

```diff
@@ -33,23 +33,15 @@ type Server struct {
 	qriNode *p2p.QriNode
 }
 
-// New creates a new qri server with optional configuration
-// calling New() with no options will return the default configuration
-// as specified in DefaultConfig
-func New(r repo.Repo, options ...func(*config.Config)) (s *Server, err error) {
-	cfg := config.DefaultConfig()
-	for _, opt := range options {
-		opt(cfg)
-	}
-
+// New creates a new qri server from a configuration
+func New(r repo.Repo, cfg *config.Config) (s *Server, err error) {
```

Comment:
Yeah I think this is the way to go. I was hoping in the long run this options pattern would help make configuration management easier, but this is just much cleaner.